From ee896bce94d266b3cdfd2731b954080e9b89932d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Sat, 17 Jan 2026 23:53:07 +0100
Subject: [PATCH] Exclude more environments

These desktop environments have their own tools to configure keyboard shortcuts.
---
 plugins/gtk/lxhotkey-gtk.desktop.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugins/gtk/lxhotkey-gtk.desktop.in b/plugins/gtk/lxhotkey-gtk.desktop.in
index 682ac8a..0b995d8 100644
--- a/plugins/gtk/lxhotkey-gtk.desktop.in
+++ b/plugins/gtk/lxhotkey-gtk.desktop.in
@@ -5,7 +5,7 @@ _Name=Setup Hot Keys
 _Comment=View and change Window Manager global shortcuts
 _Keywords=hotkey;shortcut;system;settings;
 Categories=DesktopSettings;Settings;Core;GTK;
-NotShowIn=GNOME;KDE;XFCE;MATE;
+NotShowIn=GNOME;KDE;XFCE;MATE;Budgie;COSMIC;Enlightenment;LXQt;X-Cinnamon;
 Exec=lxhotkey --gui=gtk
 StartupNotify=true
 Terminal=false
